jscomp/test/Import.res

```diff
@@ -0,0 +1,8 @@
+let each = Js.import(Belt.List.forEach)
```

Comment:
Perhaps wrap it in lazy, or add `() =>` and check that both ways it work fine.

Reply:
Just double checking: the current example loads eagerly right?
Actually not sure whether it should. What do you think?

Reply:
Oh, I'll check after adding a test of lazy evaluation with `() =>`

Reply:
I've fixed the lazy evalutation https://github.com/rescript-lang/rescript-compiler/pull/5703/commits/e8641288ad0a735d71d8305ce5846777b58eca9f